docs/core/tutorials/using-on-mac-vs.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Get started with .NET Core using Visual Studio for Mac
+title: Get started using Visual Studio for Mac
```

Comment:
I like the original here because the emphasis is on getting started with .NET Core, not VS for Mac.

Reply:
Fixed.